### PR TITLE
Tickets/dm 26502

### PR DESCRIPTION
**Pull request by teweitsai**

### CODE REVIEW

File: python/lsst/ts/ofc/ctrlIntf/OFCCalculationFactory.py [hunk 1]
```diff
@@ -51,7 +52,7 @@ def getCalculator(instName, state0Dof=None):
         Returns
         -------
         OFCCalculationOfLsst, OFCCalculationOfComCam, OFCCalculationOfSh, or
-        OFCCalculationOfCmos
+        OFCCalculationOfCmos, OFCCalculationOfLsstFam
```
Comment (pkubanek) [inline python/lsst/ts/ofc/ctrlIntf/OFCCalculationFactory.py:55]: misplaced or - shall be before OFCCalculationOfLsstFam, not before OFCCalculationOfCmos. Consider replacing the list with OFCCalculation child (OFCCalculationOfComCam).

I am not sure, but my understanding is that you shall drop OFCCalculationOfLsst class, as it is now (renamed?) OFCCalculationOfLsstFam - at least drop of LSST from InstName suggest this.

File: python/lsst/ts/ofc/ctrlIntf/OFCCalculationOfLsstFam.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        Parameters
+        ----------
+        state0Dof : dict, optional
+            State 0 DoF dictionary. If None (=default), the instrument's
+            default will be used. See
+            :lsst:ts:ofc:`OptCtrlDataDecorator.getState0FromDict` for format
+            details.
+        """
+
+        super(OFCCalculationOfLsstFam, self).__init__(
```
Comment (pkubanek) [inline python/lsst/ts/ofc/ctrlIntf/OFCCalculationOfLsstFam.py:42]: Python3 comes with slightly shorter version of super: super().__init__(, name of the class not needed.

**Review by pkubanek — CHANGES_REQUESTED** [review]
Please see Python3 super() (easier than in Python2) usage.

File: python/lsst/ts/ofc/ctrlIntf/OFCCalculationFactory.py (hunk 1 above)
Comment (teweitsai) [reply python/lsst/ts/ofc/ctrlIntf/OFCCalculationFactory.py]: Fixed in the commit: c90538b. Thanks!

File: python/lsst/ts/ofc/ctrlIntf/OFCCalculationOfLsstFam.py (hunk 2 above)
Comment (teweitsai) [reply python/lsst/ts/ofc/ctrlIntf/OFCCalculationOfLsstFam.py]: Fixed in the commit: c90538b. Thanks!

**Review by pkubanek — APPROVED** [review]

**Pull request merged**